### PR TITLE
Update prod.tfvars

### PR DESCRIPTION
**Pull request by nathan-clark**
https://tools.hmcts.net/jira/browse/RDM-10101

### Change description ###
Update D/B to 16 cores

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by MSancaktutar — APPROVED** [review]
LGTM

**Comment by hemantt** [comment]
LTGM

**Review by MSancaktutar — CHANGES_REQUESTED** [review]
Could we pls apply the same on the perftest DB?

File: infrastructure/prod.tfvars [hunk 1]
```diff
@@ -1,4 +1,4 @@
-database_sku_name     = "GP_Gen5_8"
-database_sku_capacity = "8"
+database_sku_name     = "GP_Gen5_16"
```
Comment (mario-paniccia) [inline infrastructure/prod.tfvars:1]: and perf test?

**Review by mario-paniccia — APPROVED** [review]
approved with one comment

**Review by MSancaktutar — APPROVED** [review]
LGTM

**Pull request merged**